examples/experimental/Federated Learning Experiment/server.py

```diff
@@ -23,6 +22,7 @@ def get_model():
 
 @app.route("/send_data", methods=["POST"])
 def send_data():
+    global ptr
```

Comment:
why is it global?
can't you rename ptr -> _ instead?

Reply:
I thought this line should change the ```ptr``` variable from this file (which is a global)